pnpm-lock.yaml

```diff
@@ -135,15 +135,15 @@ importers:
         specifier: ^11.1.0
         version: 11.1.0
       llamaindex:
-        specifier: workspace:*
-        version: link:../../packages/core
+        specifier: latest
+        version: link:../packages/core
```

Comment:
@yisding this is pretty cool: If we would keep this line, the `examples` package would always use the local version (when using `pnpm`).
As we might overwrite this by accident, I added the `pnpm link` command to the readme anyhow